var/spack/repos/builtin/packages/flang/package.py

```diff
@@ -18,7 +18,7 @@ class Flang(CMakePackage, CudaPackage):
 
     maintainers = ['naromero77']
 
-    version('master', branch='master')
+    version('master', branch='master', preferred=True)
```

Comment:
We generally try to avoid defaulting to moving targets like master because it can't be checksummed. Is there a good reason for this change?

Reply:
Understood about the checksum.

Yeah, the last tag was back in 20190329 and many bugs have been fixed since then. I keep receiving Spack issues that are fixed in master, but everyone installs 20190329 because it is the default. The developers have not tagged a new release in over a year -- I have spoken to them about tagging a new release. I can speak to them about it again. But mostly I did have the bandwidth to keep reply to users to do `spack install flang@master` instead.